### PR TITLE
Add a link to /find_people (with preset camp filter) from the /coordi…

### LINKED ISSUE #424: Add a link to /find_people (with preset camp filter) from the /coordinator_home page.
### Subject of the issue
Currently, it is possible for Relief Camp coordinators to register people and update requirements from `$SERVER_NAME/coordinator_home/`. The way to view all the people in the camp is via the `/find_people` page. It would be very handy for coordinators to be able to simply view all the people registered under their camps.

This is likely a low priority, but something convenient for coordinators (and a good first issue).

(I have skipped the other headers because this is not a bug, but a tiny feature request)

### PR DESCRIPTION
**Pull request by Albin-John**
…nator_home page.

### Issue Reference
This PR addresses the Issue : Fixes #424 

### Summarize
Please, describe what the PR does, the changes you have made.

> Also, mention the key points if any to look into while code reviews

### CODE REVIEW

**Comment by yeaske** [comment]
@Albin-John 
Your changes have both

> <a class="btn btn-info btn-lg" href="/find_people">
>                 Person Search - വ്യക്തിയെ തിരയുക
>         </a>

and

> <a href="/find_people">Search People</a> 

doing the same thing. Is it intended?

File: mainapp/templates/mainapp/coordinator_home.html [hunk 1]
```diff
@@ -3,11 +3,17 @@
...
-    <a href="/logout">Logout</a>
-</div>
+<div class="row">
+    <div class="col-md-4">
+        <a class="btn btn-info btn-lg" href="/find_people">
+                Person Search - വ്യക്തിയെ തിരയുക
+        </a>
+    </div>
+    <div class="text-right">
+        <a href="/find_people">Search People</a> |
```
Comment (vinitkumargoel) [inline mainapp/templates/mainapp/coordinator_home.html]: change here also or remove

Comment (ragsagar) [inline mainapp/templates/mainapp/coordinator_home.html]: Please use url name reversal instead of hardcoding.

**Comment by remedcu** [comment]
This can be closed @syam3526 @biswaz and @vigneshhari as the problem is already solved:

`<a href="{%url 'find_people' %}">Search People</a>`

**Pull request closed without merging**